include/globals.h

```diff
@@ -730,12 +730,12 @@ extern RemoteDebug Debug;           // Let everyone in the project know about it
     #define PROJECT_NAME            "Ledstrip"
     #endif
 
-    #define ENABLE_WEBSERVER            1   // Turn on the internal webserver
```

Comment:
Make more memory available in LEDSTRIP/DEMO builds

Reply:
So the webserver is no longer  able to be used on LEDSTRIP builds  on a standard  esp32 with 4M memory?

I've been trying to get mine to work and it won't launch.